### DISCUSSION TITLE
How will the overall theme be?

### DISCUSSION THREAD

**Discussion by Bitwolfies** [jellyfin/jellyfin-vue#301] [UI & UX]
(This new discussion feature is fantastic, finally I can ask questions without feeling like a total ass for opening an issue.)

I suppose it’s a bit early to be asking about themes, but I was wondering about how if what we see now is basically how it will look in the end colour wise. Honestly I really love the Jellyfin gradient™ and it’s a shame that most clients don’t take advantage of it. Just wondering about the plans for that sort of thing, thank you. :)

**Comment by heyhippari** [comment]
There hasn't been much thought on the overall theme yet aside from layout, unfortunately.

I've tried out a few color schemes, but I haven't found anything really suitable that works with the Jellyfin colors, either. As for the gradient colors, they're generally too bright for them to work well in a UI, in my opinion. I do want to imply them in some way, perhaps with a similar hue but a different lightness?

But yeah, if anyone reading this has ideas, we're open to proposals :)

**Reply by Bitwolfies** [reply]
Right, that makes sense. I suppose I'm just a huge fan of I guess you'd call it the swift ui/Windows 10 style? Lots of clear translucent menus and bars with gradients behind it. Currently in jellyfin I use the blue radiance theme, with css to add translucent elements. Which is the way I hope vue goes (or at least can be customized to.)
